moveit_runtime/package.xml

```diff
@@ -1,18 +1,20 @@
-<?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<!-- This XML code describes a ROS package named moveit_runtime -->
+
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd"?>
+
 <package format="3">
   <name>moveit_runtime</name>
   <version>2.7.1</version>
   <description>moveit_runtime meta package contains MoveIt packages that are essential for its runtime (e.g. running MoveIt on robots).</description>
-  <maintainer email="henningkayser@picknik.ai">Henning Kayser</maintainer>
-  <maintainer email="tyler@picknik.ai">Tyler Weaver</maintainer>
+  <maintainer email="example_maintainer1@example.com">Henning Kayser</maintainer>
+  <maintainer email="example_maintainer2@example.com">Tyler Weaver</maintainer>
 
   <license>BSD-3-Clause</license>
 
   <url type="website">http://moveit.ros.org</url>
   <url type="website">http://wiki.ros.org/moveit_runtime</url>
 
-  <author email="gm130s@gmail.com">Isaac I. Y. Saito</author>
+  <author email="example_author@example.com">Isaac I. Y. Saito</author>
```

Comment:
Don't you think Henning, Tyler, and Isaac would remove the emails themselves if they wanted them removed? I think this PR should be closed.

Reply:
GMail thought I am pinged in this PR. Turned out I am :)

Since this is about moveit2, not 1, technically I haven't contributed anything (yet), so removing authorship of mine is actually appropriate.

Having said that, I don't understand the rationale of everything else done in this PR, unless I'm missing recent trend (replacing email with a fake address? Removing author tag of those who actually contributed?).